src/BabylonCpp/src/asio/asio.cpp

```diff
@@ -184,10 +185,15 @@ static std::string ArrayBufferToString(const ArrayBuffer & dataUint8)
   return dataString;
 }
 
-bool HACK_DISABLE_ASYNC = false;
-void set_HACK_DISABLE_ASYNC(bool v)
+unsigned int HACK_DISABLE_ASYNC = 0;
+void push_HACK_DISABLE_ASYNC()
 {
-  HACK_DISABLE_ASYNC = v;
+  ++HACK_DISABLE_ASYNC;
+}
+void pop_HACK_DISABLE_ASYNC()
+{
+  --HACK_DISABLE_ASYNC;
+  assert(HACK_DISABLE_ASYNC >= 0);
```

Comment:
oops, this test can not fail with an unsigned int!  The compiler did not even blink...